### ISSUE TITLE
UI: aggregate logs from arbitrary selected resources

### ISSUE THREAD

**Issue by palra** [tilt-dev/tilt#6127]
*Foreword: I've managed to onboard my company into using Tilt in order to manage our local development environment, and I have to say this is one of the best technical choices we ever made! I truly love Tilt and I wanted to thank the Tilt team and community for this awesome project. 🙌🏾*

## Describe the Feature You Want
I'd like to be able to stream logs from multiple specific resources instead of having to switch from one to the other. Ideally, I'd like to stream logs that are under the same label.

### Current Behavior
Right now, we can see the logs of a single resource or stream all of the resources logs at the same time.

### Why Do You Want This?
When debugging some issue across, I often have to follow the path of a query across multiple services. Having multiple debugger instances isn't always the easiest for this, quickly printing some debug logs might sometimes be preferable. I can display the logs for all of the resources but the logs I want to see are often lost in a sea of unrelated debug traces from other services. I would like to have the "All Resources" view but only for some selected resources in order to filter out the useless containers in my debugging session.

**Additional context**
_N/A_

**Comment by nicks** [comment]
You can currently star resources, then view them together in the starred panel. Does that help?

**Comment by palra** [comment]
You're right, that helps! Is it a workaround or was it actually designed for this specific use case? I never noticed the stars before, my bad.
